active/fellowship.md

```diff
@@ -2,30 +2,62 @@
 
 ## Scope of responsibilities
 
-The Fellowship Working Group manages the operation of the [Django Fellowship Program](https://www.djangoproject.com/fundraising/#fellowship-program). They select candidates to fill the fellowship, and act as line managers during the operation of the program.
+The Fellowship Working Group manages the [Django Fellowship Program](https://www.djangoproject.com/fundraising/#fellowship-program). This includes:
+
+- Interviewing candidates for open Fellowship slots and making hiring recommendations to the Board. This includes developing and maintaining interview and selection scripts, exercises, and rubrics.
+- Acting as an HR committee for Fellows. In the event of a complaint about a Fellow, or if there are performance issues, the Fellowship-WG will investigate, in concert with the [Code of Conduct WG](code-of-conduct.md) if necessary, and present findings and a recommendation to the board for action.
+  - As with the CoC-WG, the Fellowing-WG has the power to unilaterally suspend Fellows, with pay, pending an investigation. This power should only be used in egregious circumstances, where waiting on investigation and/or board action might plausibly be harmful.
+- Designating one member of the committee to act as the line manager for active fellows.
+  - The Board must approve this designation.
+  - Defining what it means to be a manager is out of scope for this document, but it's all the typical stuff: one-on-ones, feedback, coaching, and performance management, if necessary.
+  - It's expected that this person will have previous engineering management experience.
+  - This person also approves expenses for Fellows, subject to the [budget](#budget), see below.
+  - Perform bi-annual performance reviews for the fellows, and reports to the board.
+    - This can be super informal, short and sweet is fine. This is intended to be a backstop against the unlikely event of performance issues with a fellow, not any sort of stack ranking etc.
+- Acting as an escalation point if a Fellow has a disagreement with their designated manager. That is, the escalation path for Fellows is: designated manager -> full Fellowship WG -> Board.
+- Maintaining the formal job description and outline of duties for Fellows. Those changes must be reported to the Board who may veto them by board vote.
```

Comment:
Could we look at this information as part of reviewing this WG charter? I’m not sure where those are defined currently.

Reply:
They're not - my intent was to clarify that the WG needs to have them. I'll clarify that we expect the first group under this new charter to create the first versions of these!